templates/web.letsencrypt.ssl.template.yml

```diff
@@ -19,6 +19,20 @@ run:
         LE_WORKING_DIR="${LETSENCRYPT_DIR}" ./acme.sh --upgrade --auto-upgrade
         LE_WORKING_DIR="${LETSENCRYPT_DIR}" ./acme.sh --set-default-ca  --server  letsencrypt
 
+        cat << EOF > /etc/nginx/conf.d/outlets/before-server/20-redirect-http-to-https.conf
```

Comment:
In, for example, `samples/web_only.yml`, we tell operators to uncomment both `web.ssl.template.yml` and `web.letsencrypt.ssl.template.yml`.

However, the `configure-letsencrypt` and `configure-ssl` scripts are writing to the same outlet. So future changes to the latter are going to be mysteriously overwritten by the former.